release_notes/ocp-4-5-release-notes.adoc

```diff
@@ -149,6 +148,19 @@ You can now use CSI to specify volumes directly in the Pod specification, rather
 
 Volume cloning using CSI, previously in Technology Preview, is now fully supported in OpenShift Container Platform 4.5. For more information, see xref:../storage/container_storage_interface/persistent-storage-csi-cloning.adoc#persistent-storage-csi-cloning[CSI volume cloning].
 
+[id="ocp-4-5-olm"]
+=== Operator Lifecycle Manager
+
+[id="ocp-4-5-olm-v1-crd"]
+==== v1 CRD support
+
+Operator Lifecycle Manager (OLM) now supports Operators using v1 Custom Resource
+Definitions (CRDs) in catalogs and on cluster. Previously, OLM only supported
+v1beta1 CRDs when loading Operators into catalogs and deploying them on cluster.
+
+To support this feature, OLM also now supports safely deprecating and removing
+older storage versions of CRDs to ensure there is no data loss.
```

Comment:
So we ended up just doing a minimal fix for this - by blocking an upgrade if a stored CRD version on-cluster is missing in the spec of the new CRD. I wouldn't say that "we support safely deprecating" but more that "OLM now enforces CRD upgrades are safer by ensuring existing CRD storage versions are not missing in the upgraded CRD and potentially causing data loss".